framework/common/include/mapf/common/encryption.h

```diff
@@ -71,6 +79,11 @@ class diffie_hellman {
      */
     unsigned nonce_length() const { return sizeof(m_nonce); }
 
+    /**
+     * @brief Maximum length of pubkey().
+     */
+    static const unsigned max_pubkey_length = 192;
```

Comment:
I'm confused - It looks like the pubkey length is exactly 192 with the new logic which is added. Shouldn't we change the terminology to always treat the pubkey as 192 bytes? (nitpick)

Reply:
The pubkey can be smaller (i.e. pubkey_length() can be smaller). But copy_pubkey always writes to a 192-byte buffer.